infrastructure-pr-testing.markdown

```diff
@@ -38,15 +38,15 @@ Checkers on macOS do not use Aurora, instead they are run directly as scripts.
  interface](http://aurora.apache.org/documentation/0.16.0/reference/client-commands/)** and
 make sure **you set up your [ALICE Aurora environment](https://alisw.github.io/infrastructure-aurora) correctly**.
 
-* [Setup your environment](#setup)
-* [Listing active PR checker](#list-checkers)
-* [Deploying a PR checker](#deploy-checker)
-* [Scaling up the number of checkers](#scale-up-checkers)
-* [Scaling down the number of checkers](#scale-down-checkers)
-* [Restart a PR checker](#restart-checker)
-* [Removing a PR checker](#remove-checker)
-* [Inspecting the status of the checker](#inspect-checker)
-* [Monitoring the checkers](#monitor-checkers)
+* [Setup your environment](#setup-your-environment)
+* [Listing active PR checker](#listing-active-pr-checkers)
+* [Scaling up the number of checkers](#scaling-up-the-number-of-checkers)
+* [Scaling down the number of checkers](#scaling-down-the-number-of-checkers)
+* [Creating a new checker](#creating-a-new-checker)
+* [Updating a PR checker](#updating-a-pr-checker)
+* [Restarting a checker](#restarting-a-checker)
+* [Inspecting the checkers](#inspecting-the-checkers)
+* [Monitoring the checkers](#monitoring-the-checkers)
```

Comment:
If you want to use the automatically generated ones it's fine, however could you also remove the labels I added by hand (the bits in the form `{:something}`)

Reply:
The links via labels do not work, that's why I changed them into the "raw" ones. Ok for removing the labels.

Reply:
They must have changed it. Ok. Merging.